packages/api/cms-api/src/dam/files/files.controller.ts

```diff
@@ -90,6 +94,11 @@ export function createFilesController({ Scope: PassedScope }: { Scope?: Type<Dam
             @GetCurrentUser() user: CurrentUser,
             @Headers("range") range?: string,
         ): Promise<void> {
+            if (process.env.NODE_ENV) {
```

Comment:
This check isn't right, because NODE_ENV is also set in production environments. Should this be `process.env.NODE_ENV === "development"`?